.changeset/fuzzy-terms-push.md

```diff
@@ -0,0 +1,35 @@
+---
+'@web/browser-logs': minor
+'@web/config-loader': minor
+'@web/dev-server': minor
+'@web/dev-server-core': minor
+'@web/dev-server-esbuild': minor
+'@web/dev-server-hmr': minor
+'@web/dev-server-import-maps': minor
+'@web/dev-server-legacy': major
+'@web/dev-server-rollup': minor
+'@web/dev-server-storybook': minor
+'@web/parse5-utils': major
+'@web/polyfills-loader': major
+'@web/rollup-plugin-copy': minor
+'@web/rollup-plugin-html': major
+'@web/rollup-plugin-import-meta-assets': major
+'@web/rollup-plugin-polyfills-loader': major
+'@web/test-runner': minor
+'@web/test-runner-browserstack': minor
+'@web/test-runner-chrome': minor
+'@web/test-runner-cli': minor
+'@web/test-runner-commands': minor
+'@web/test-runner-core': minor
+'@web/test-runner-coverage-v8': minor
+'@web/test-runner-junit-reporter': minor
+'@web/test-runner-mocha': minor
+'@web/test-runner-playwright': minor
+'@web/test-runner-puppeteer': minor
+'@web/test-runner-saucelabs': minor
+'@web/test-runner-selenium': minor
+'@web/test-runner-visual-regression': minor
+'@web/test-runner-webdriver': minor
+---
+
+Set node 16 as the minimum version.
```

Comment:
I'm probably missing something, but why should it be included in this list?

Reply:
The reason to include it would be that the GitHub Action no longer including 14 effects ALL packages, even though it isn't directly in their directory tree. Nothing supports 14 any more, so everything is breaking.

The reason not to include it would be that while the above is true, there don't seem to be any tests for this package that would have run in 14, so we never technically supported 14 to begin with?

Reply:
I think bumping it then makes sense. If not, just to make sure, we break something in an unrelated version bump later on.